### ISSUE TITLE
Stat bonus when level up from accessories not working on Maya.

### ISSUE THREAD

**Issue by aradovn** [sayucchin/P2-EP-PSP#10]
Accessories that bonus stat when level up like Strength source, Lucky source, Agility source...not working on Maya. I did test and find out that if play same save on original Japan ISO then it still work. 
 Other party member not affect by this. I can also provide save file before level up if need.

**Comment by sayucchin** [comment]
Yeah it's an annoying visual glitch. Take screenshots and you'll see that Maya did get the stat point. If you equip a persona with a matching bonus stat with the accessory then you'll see a +2 in the bonus slot, but if they're mismatched the persona stat is in the bonus box and the accessory stat goes in the bottom box and is visible for the other party members but because Maya doesn't have fixed stats for each level up we couldn't get it to show up in that box for her.

**Comment by aradovn** [comment]
So 
![Screenshot_20221118-142621_PPSSPP_Gold](https://user-images.githubusercontent.com/86295977/202648544-b76ac80f-6cc3-40ac-9397-df539be99722.png)
I do as you said and go to test: equip Maya with Lucky Source and persona Sarutobi that +LCK and this is what happen: Maya default luck is 3, it go to 4, the box say +1.

Also if I do that in no patched ISO:
![Screenshot_20221118-145003_PPSSPP_Gold](https://user-images.githubusercontent.com/86295977/202649679-da7fa0ad-a518-4009-94af-469b3e746c5d.png)

**Comment by sayucchin** [comment]
Oh interesting I thought we had fixed that. Mind sending me your save file+system save? My email is sayucchi.msg@gmail.com

**Comment by aradovn** [comment]
No problem, I'll send right away. Thanks for your great work!

**Comment by sayucchin** [comment]
Okay I had a look at it and you were right. It's just Maya that is bugged for the stat increasing accessories. Hp and sp source work just fine but not the others. And it wasn't broken in the jp version. We'll have to investigate and hopefully we'll find a fix.

**Comment by cmccord-dev** [comment]
hey sorry it took a bit but we figured out what was happening!

**Comment by aradovn** [comment]
Nice, I will do second playthrough when it got fixed.

**Comment by Erquint** [comment]
Was the patch fixed since?
The issue seems to remain open.

**Comment by sayucchin** [comment]
no we're not doing incremental updates. We're giving people time to discover bugs and once we have time and energy we'll release a final update that'll hopefully fix any issues and close the project.
